### PR TITLE
Readme finished

### PR DESCRIPTION
**Pull request by Zaretsky-Daniel**
The main readme file is finished. Now I will start working on the examples and the personalization files.
In the Readme file at rows 334 and 338 there is a need to add the correct "http" address of the files when they are pulled into the main.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -224,28 +224,118 @@ particlePositions(:,:,2) = particlePositionsY;
...
+		plot(currStepData.trapPositions(1,i),currStepData.trapPositions(2,i),'o'); 
+    end
+    pause(1); 
+```
+Using the limits we set in the configuration file we plot all of the traps current places.
 
 ## Basic examples
 
-Please read [Examples.md](https://gist.github.com/PurpleBooth/b24679402957c63ec426) for some basic usage examples
+Please read [Examples.md](http) for some basic usage examples
```
Comment (Zaretsky-Daniel) [inline README.md:334]: Here please change the link to the relevant one.

File: README.md [hunk 2]
```diff
@@ -224,28 +224,118 @@ particlePositions(:,:,2) = particlePositionsY;
...
 
 ## How to personalize
 
-Please read [Personalization.md](https://gist.github.com/PurpleBooth/b24679402957c63ec426) for details about how to add new forces or other functions.
+Please read [Personalization.md](http) for details about how to add new forces or other functions.
```
Comment (Zaretsky-Daniel) [inline README.md:338]: Also here please change the link to the relevant one.

File: README.md (hunk 1 above)
Comment (giladpollack) [reply README.md:334]: Not sure what you mean, since as far as I know there isn't a relevant one yet at the moment

**Comment by giladpollack** [comment]
I'm not sure what exactly needs to be in the link, so either do it if it's simple or explain to me what you mean =)

**Pull request merged**